Test/dafny2/Z-BirthdayBook.dfy.expect

```diff
@@ -1,3 +1,3 @@
 
 Dafny program verifier finished with 21 verified, 0 errors
-Send cards to: {Mike, Susan}
+Send cards to: {['M', 'i', 'k', 'e'], ['S', 'u', 's', 'a', 'n']}
```

Comment:
This is a little weirder than the character instance. Is this the best way to print strings? Does it only come about due to refinement?

Reply:
See other answer, same root cause. Nothing to do with refinement at least.

Reply:
It seems to me that other tests were printing strings like "Mike" (adding double quotes), which seems like a nice behavior to me. Was I misinterpreting that, or is something different going on here?

Reply:
It's only because this is the result of (effectively) `print {"Mike", "Susan"};`. So it's printing a `set<string>` rather that `string` values. The runtimes COULD handle this better, but they'd need to accept a `areElementsStrings` parameter which the compiler would fill in when it knows the target `set<T>` type is in fact a `set<string>`.